style.css

```diff
@@ -51,6 +52,79 @@
     }
   }
 
+  .hamburger {
+    display: block;
+    cursor: pointer;
+    width: 30px;
+    height: 25px;
+    position: relative;
+    left: 78vw;
+    top: 0.5;
+    appearance: none;
+    outline: none;
+    background: none;
+    border: none;
+    z-index: 4;
+  }
+
+  .hamburger .bar,
+  .hamburger::after,
+  .hamburger::before {
+    content: "";
+    display: block;
+    width: 100%;
+    height: 2.5px;
+    background: #6070ff;
+    margin: 4px 0;
+    transition: 0.4s;
+    border-radius: 2px;
+  }
+
+  .hamburger.is-active::before {
+    transform: rotate(-45deg) translate(-2.5px, 2px);
+    background-color: #fff;
+  }
+
+  .hamburger.is-active::after {
+    transform: rotate(45deg) translate(-2.5px, -2px);
+    background-color: #fff;
+  }
+
+  .hamburger.is-active .bar {
+    display: none;
+  }
+
+  .mobile-nav {
+    position: fixed;
+    left: 100%;
+    transition: 2s;
+  }
+
+  .mobile-nav.is-active {
+    display: flex;
+    flex-direction: column;
+    left: 0;
+    background: #6070ff;
+    width: 100vw;
+    height: 100vh;
+    margin-top: -1.5rem;
+    padding-top: 4rem;
+    overflow-y: hidden;
+    z-index: 1;
+    opacity: 90%;
+    backdrop-filter: blur(8px);
```

Comment:
- [ ]  Please increase your background blurring and remove opacity to match the figma design:
>yours: 
![image](https://user-images.githubusercontent.com/91486791/223481040-75e8be1f-a65e-4143-ad81-771e738684e3.png)
>figma:
![image](https://user-images.githubusercontent.com/91486791/223481821-0a156670-04b7-440b-aa32-026662772a67.png)